front/scripts/api-errors-i18n.ts

```diff
@@ -0,0 +1,88 @@
+import SwaggerParser from '@apidevtools/swagger-parser';
+import i18next from 'i18next';
+import { noop } from 'lodash';
+
+// relative the project's root
+const openapi_path = '../editoast/openapi.yaml';
+// relative to this file
+const i18n_error_path = '../public/locales/fr/errors.json';
+
+/**
+ * Check if the given error is well filled in the i18n.
+ */
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+async function checkI18N(error: any): Promise<string[]> {
+  const result: string[] = [];
+
+  // Error data for i18n
+  const errorName = error.properties.type.enum[0];
+  const errorId = errorName.split(':').join('.');
+  const errorVarExample = Object.keys(error.properties.context.properties || {}).reduce(
+    (acc, curr) => {
+      acc[curr] = '';
+      return acc;
+    },
+    {} as Record<string, string>
+  );
+
+  // Init the i18n system
+  const i18nData = require(i18n_error_path);
+  const i18n = await i18next.createInstance(
+    {
+      lng: 'en',
+      resources: {
+        en: {
+          translation: i18nData,
```

Comment:
Is it normal that the path refers to the fr json but the prop is `en` here ?

Reply:
Ho good catch, I changed from `en` to `fr`, because the default lang for the app is `fr`.
Even if it works like that, it's better to put `fr` everywhere.